scripts/check-tags.ts

```diff
@@ -9,27 +9,38 @@ interface Project {
   tags: string[]
 }
 
-const checkTags = async (filePath: string) => {
+const hasValidTags = async (filePath: string) => {
   const projectText = await readFile(filePath, 'utf8')
   const project = JSON.parse(projectText) as Project
   const tagsRaw = project.tags ?? []
   const tags = tagsRaw.map((tag) => tag.split(':'))
-  const isNormal =
-    (tags.length === 1 && tags[0][0] === 'scope') ||
-    (tags.length === 2 &&
-      tags.find(([key]) => key === 'scope') &&
-      tags.find(([key]) => key === 'lib') &&
-      tags[0][1] === tags[0][1])
+
   const isEmpty = tags.length === 0
+  // Only allow 'scope:*' or 'lib:*' tags
+  const validPrefix = tags.every(([key]) => key === 'scope' || key === 'lib')
+  // Must have at least one scope tag or be empty
+  const hasScopePrefix = tags.some(([key]) => key === 'scope') || isEmpty
+  // Are all tags the same?
+  const singularTag = new Set(tags.map(([, value]) => value)).size === 1
```

Comment:
~This allows eg `scope:react lib:react lib:react` which is not "expected" but not a big issue.

Reply:
Added `repeatFreeTags` check